widgets-range/README.md

```diff
@@ -1,14 +1,14 @@
 ## Example: Histogram widget
 
-A simple example that showcases how to build a scalable [Histogram widget](http://todo.com) using CARTO + deck.gl that stays synchronized with the map, using large-scale data coming from the same cloud data warehouse as your maps.
+A simple example that showcases how to build a scalable [Range widget](https://docs.carto.com/carto-for-developers/reference/carto-widgets-reference/models/gethistogram) using CARTO + deck.gl that stays synchronized with the map, using large-scale data coming from the same cloud data warehouse as your maps.
```

Comment:
Minor - Text and link don't match here, "range widget" vs "githistogram". Problem appears here and in several other `widgets-*/README.md` files.

Reply:
Ugh, good catch – So many files. Fixing